src/helpers/tests/parser.test.ts

```diff
@@ -33,7 +33,7 @@ describe("generateFileString", () => {
 
   test("generate augmented file string success (js)", async () => {
     setupFolderStructure("./src/models", { js: true, augment: true });
-    const modelsPath = await paths.getFullModelsPaths(".", "js");
+    const modelsPath = await paths.getFullModelsPaths("", "js");
```

Comment:
Previously "." was testing the behaviour if a user did not provide a model folder path (default to "."). With new changes this would be perceived as user input and search for Mongoose files at only that path. To test the new default, we can pass either undefined or an empty string, ensuring folders will be searched recursively for `models/` folder.